src/libraries/System.Text.Json/tests/System.Text.Json.Tests/System.Text.Json.Tests.csproj

```diff
@@ -4,6 +4,9 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- SYSLIB0020: JsonSerializerOptions.IgnoreNullValues is obsolete -->
     <NoWarn>$(NoWarn);SYSLIB0020</NoWarn>
+
+    <!-- these tests depend on the pdb files -->
```

Comment:
Really? Do you know where @layomia @steveharter

Reply:
[`437d614` (#51697)](https://github.com/dotnet/runtime/pull/51697/commits/437d614dc33b98d353eabf50b18f2528e9db1bca)

Debugger tests:
```
    fail: [FAIL] System.Text.Json.Tests.DebuggerTests.DefaultJsonElement
    info: System.NullReferenceException : Object reference not set to an instance of an object.
    info:    at System.Text.Json.Tests.DebuggerTests.GetDebuggerDisplayProperty[JsonElement](JsonElement value)
    info:    at System.Text.Json.Tests.DebuggerTests.DefaultJsonElement()
    info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
    fail: [FAIL] System.Text.Json.Tests.DebuggerTests.DefaultUtf8JsonWriter
    info: System.NullReferenceException : Object reference not set to an instance of an object.
    info:    at System.Text.Json.Tests.DebuggerTests.GetDebuggerDisplayProperty[Utf8JsonWriter](Utf8JsonWriter value)
    info:    at System.Text.Json.Tests.DebuggerTests.DefaultUtf8JsonWriter()
    info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
    fail: [FAIL] System.Text.Json.Tests.DebuggerTests.DefaultJsonProperty
    info: System.NullReferenceException : Object reference not set to an instance of an object.
    info:    at System.Text.Json.Tests.DebuggerTests.GetDebuggerDisplayProperty[JsonProperty](JsonProperty value)
    info:    at System.Text.Json.Tests.DebuggerTests.DefaultJsonProperty()
    info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
```

Reply:
This one could be fixed in the test pretty easily - 

```C#
        private static string GetDebuggerDisplayProperty<[DynamicallyAccessedMembers.PublicProperties] T>(T value)
        {
            return (string)typeof(T).GetProperty("DebuggerDisplay", BindingFlags.Instance | BindingFlags.NonPublic).GetValue(value);
        }
```